setup.py

```diff
@@ -86,36 +86,9 @@
         "Compile fonts from sources (UFO, Glyphs) to binary (OpenType, TrueType)."
     ),
     long_description=long_description,
-    author=(
-        "fontmake authors and contributors:"
-        " James Godfrey-kittle,"
-        " Cosimo Lupo ,"
-        " Felipe Sanches,"
-        " Nikolaus Waxweiler,"
-        " Yanone ,"
-        " Behdad Esfahbod,"
-        "Jany Belluz ,"
-        " Khaled Hosny,"
-        " Just Van Rossum,"
-        " Marc Foley,"
-        "Dave Crossland ,"
-        "Misha Brukman,"
-        "Pyup-bot, "
-        "Dennis Moyogo Jacquerye,"
-        "Simon Cozens,"
-        "NFSL2001,"
-        "Bo Song,"
-        "Sascha Brawer,"
-        "Antonio Cavedoni,"
-        "Adam Twardoch,"
-        "Fabian Greffrath,"
-        "Thomas Rettig,"
-        "Chris Simpkins,"
-        "Adrien Tetar"
-    ),
-
-
-    
+    author="James Godfrey-kittle",
+    maintainer=" Cosimo Lupo",
+   
```

Comment:
please don't leave the empty line or black auto-formatter is going to bark at you.
Also you can add 

```
maintainer_email="cosimo@anthrotype.com",
```